bills-app/src/main/webapp/script.js

```diff
@@ -26,3 +26,19 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function addCode() {
+    getBillData();
+}
+
+function getBillData() {
```

Comment:
```suggestion
    getUpcomingBillsData();
}

function getUpcomingBillsData() {
```
nit: just to make it super clear what bills these are because in the future we may choose to also retrieve recently passed bills, bills filtered by a specific keyword, etc.